tests/lib/rules/id-length.js

```diff
@@ -66,7 +66,10 @@ ruleTester.run("id-length", rule, {
         { code: "export var num = 0;", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
         { code: "import * as something from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
         { code: "import { x } from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
+        { code: "import { x as x } from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
```

Comment:
```suggestion
        { code: "import { x as x } from 'y';", languageOptions: { ecmaVersion: 6, sourceType: "module" } },
        { code: "import { 'x' as x } from 'y';", languageOptions: { ecmaVersion: 2022, sourceType: "module" } },
```

It would be good to add a test with a string literal import name.